lib/client/chart.js

```diff
@@ -43,6 +43,11 @@ function init (client, d3, $) {
   var yScale2 = chart.yScale2 = d3.scale.log()
     .domain([utils.scaleMgdl(36), utils.scaleMgdl(420)]);
 
+  var xScaleBasals = chart.xScaleBasals = d3.time.scale().domain(extent);
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['xScaleBasals' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/1129544652/issues/source?bid=2452379&fileBranchId=2886508#l46)